languages/elixir/exercises/concept/conditionals/lib/log_level.ex

```diff
@@ -0,0 +1,3 @@
+defmodule LogLevel do
+  # TODO: implement the to_label/1 function
```

Comment:
I would suggest to add a method stub here. We didn't use stubs for the basics exercise as we wanted students to learn about defining methods from scratch, but as we know students will have completed the `basics` exercise at this point, I would suggest to put in a stub here. This will allow a student to focus on the actual concept instead of the routine task of defining the method themselves. It also helps the students that will be using the online editor.

Stubs should ideally give be defined such that the student will get a nice and descriptive error message. Usually, this means that the stub should be valid, compiling code, as compiler errors can sometimes be tricky to parse for students. How this is done is different per language. See [this F# example](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/strings/Strings.fs) for one example.

Reply:
I can do that and then have it raise a an error with a relevant message

Reply:
Awesome!